resources/views/layouts/_footer.blade.php

```diff
@@ -7,6 +7,7 @@
             </div>
         </div>
     </div>
-    <cookie-law policy-url=""></cookie-law>
+
+    <privacy-info privacy-policy-url="{{ url(option('general', 'cookies_policy_url') || '') }}"></privacy-info>
```

Comment:
I think that the `option` helper has the `$fallback` variable that we can use here instead of `or` condition